src/components/overflow-menu-button/ohri-overflow-menu.component.tsx

```diff
@@ -127,6 +127,7 @@ export const OHRIOverflowMenu: React.FC<OverflowMenuProps> = ({
           {menuTitle}
         </Button>
       )}
+      )
```

Comment:
This adds a trailing bracket near the button overflow menu and i think it should be removed